docs/_includes/header_custom.html

```diff
@@ -1,2 +1,11 @@
 <link rel="stylesheet" href="{% link assets/css/main.css %}">
 
+<!-- User Engagement Survey Banner -->
+<div id="survey-banner">
```

Comment:
Not a blocker, the header shouldn't contain visual elements:

> The <head> element is a container for metadata (data about data) and is placed between the <html> tag and the <body> tag.
> Metadata is data about the HTML document. Metadata is not displayed.
> Metadata typically define the document title, character set, styles, scripts, and other meta information.
> https://www.w3schools.com/tags/tag_head.asp

> The <body> tag defines the document's body.
> The <body> element contains all the contents of an HTML document, such as headings, paragraphs, images, hyperlinks, tables, lists, etc.
> https://www.w3schools.com/tags/tag_body.asp

I wonder if there's an alterantive place to add this so it honors HTML semantics